x-pack/plugins/security_solution/server/lib/detection_engine/signals/build_signal.test.ts

```diff
@@ -124,31 +136,39 @@ describe('buildSignal', () => {
       },
       status: 'open',
       rule: {
+        author: [],
+        id: '7a7065d7-6e8b-4aae-8d20-c93613dec9f9',
+        created_at: new Date(ANCHOR_DATE).toISOString(),
+        updated_at: new Date(ANCHOR_DATE).toISOString(),
         created_by: 'elastic',
-        description: 'Detecting root and admin users',
+        description: 'some description',
         enabled: true,
-        false_positives: [],
+        false_positives: ['false positive 1', 'false positive 2'],
         from: 'now-6m',
-        id: '04128c15-0d1b-4716-a4c5-46997ac7f3bd',
         immutable: false,
-        index: ['auditbeat-*', 'filebeat-*', 'packetbeat-*', 'winlogbeat-*'],
-        interval: '5m',
-        risk_score: 50,
-        rule_id: 'rule-1',
-        language: 'kuery',
-        max_signals: 100,
-        name: 'Detect Root/Admin Users',
-        output_index: '.siem-signals',
+        name: 'Query with a rule id',
         query: 'user.name: root or user.name: admin',
-        references: ['http://www.example.com', 'https://ww.example.com'],
+        references: ['test 1', 'test 2'],
         severity: 'high',
-        updated_by: 'elastic',
+        severity_mapping: [],
+        updated_by: 'elastic_kibana',
         tags: ['some fake tag 1', 'some fake tag 2'],
         to: 'now',
         type: 'query',
-        note: '',
-        updated_at: signal.rule.updated_at,
-        created_at: signal.rule.created_at,
+        threat: [],
+        version: 1,
+        status: 'succeeded',
+        status_date: '2020-02-22T16:47:50.047Z',
+        last_success_at: '2020-02-22T16:47:50.047Z',
+        last_success_message: 'succeeded',
+        output_index: '.siem-signals-hassanabad-frank-default',
```

Comment:
nit:
```ts
.siem-signals-default
```

That would probably be better here.

Reply:
...I have no idea how that got in there.